docs/layers/monitoring/datadog/datadog.mdx

```diff
@@ -4,12 +4,18 @@ sidebar_label: "Datadog"
 sidebar_position: 10
 description: Monitor everything with Datadog
 ---
-import Intro from '@site/src/components/Intro';
-import KeyPoints from '@site/src/components/KeyPoints';
+import Steps from '@site/src/components/Steps';
+import Admonition from '@theme/Admonition';
 
 Cloud Posse advises customers that all have <i>very similar monitoring requirements</i> because we filter for customers who match our delivery model. The actual fine-tuning of it will be specific to the customer.
```

Comment:
```suggestion

<Intro>
  This outlines Cloud Posse's practical approach to monitoring with Datadog, focusing on defining reusable Service Level Indicators (SLIs) and Service Level Objectives (SLOs) for consistent implementation across customer environments. It aims to help businesses streamline monitoring and incident management by aligning technical performance with business goals.
</Intro>

<Note>
  Cloud Posse advises customers that all have <i>very similar monitoring requirements</i> because we filter for customers who match our delivery model. The actual fine-tuning of it will be specific to the customer.
</Note>
```

Reply:
Note, I deleted the part about ...

> Cloud Posse works with customers who share similar monitoring needs, aligning with our standardized delivery model. We customize the final implementation to fit the unique requirements of each customer.

As I don't feel it belongs in the public docs